### ISSUE TITLE
plotting.view_markers with different markers size

### ISSUE THREAD

**Issue by salma1601** [nilearn/nilearn#1897]
Hi all,

Is there a way to view markers with different sizes in a 3D plot ? in `plotting.view_markers`, it looks like the `marker_size` must be the same for all markers

**Comment by jeromedockes** [comment]
have you tried passing a list for marker size? it seems to work for me:
```
  from nilearn import plotting   
  dmn_coords = [(0, -52, 18), (-46, -68, 32), (46, -68, 32), (1, 50, -5)]  
  view = plotting.view_markers(
    dmn_coords, ['red', 'cyan', 'magenta', 'orange'], marker_size=[10, 20, 40, 80])  
  view.open_in_browser()  
 ```

**Comment by jeromedockes** [comment]
![newplot 1](https://user-images.githubusercontent.com/9196501/51171716-e6162600-18b1-11e9-9925-0198dfd33bcb.png)

**Comment by salma1601** [comment]
Ok sorry! I was mislead by the doc saying it is a float.
Thank you very much !

**Comment by jeromedockes** [comment]
> Ok sorry! I was mislead by the doc saying it is a float.
> Thank you very much !

thanks! actually I don't think we had using different sizes in mind, I hadn't
tried it before and indeed the documentation doesn't mention it. We should
probably convert the marker_size to a list when it is a numpy array, add a test
and update the documentation.

**Comment by jeromedockes** [comment]
#1899

**Comment by jeromedockes** [comment]
@kchawla-pi I guess we can close this as it was addressed in #1899

**Comment by kchawla-pi** [comment]
I'll check out the PR once I'm back and then close this after merging that
one

**Comment by Ysoni11** [comment]
Hello,
I have a little naive question:
the markers size parameters for the sphere is in mm ? pixel ? is it the radius ?
(I hope i didn't miss it in the doc)
Thank you

**Comment by kchawla-pi** [comment]
@jeromedockes We should add the unit of the size in the docstring.

**Comment by jeromedockes** [comment]
it is in supposed to be pixels. it is passed as is to plotly. https://plot.ly/javascript/reference/#scatter3d-marker-size .
it is just for visualization and doesn't have a particular meaning, user can try a few values until they get the aspect they want.

**Comment by kchawla-pi** [comment]
Sure, MATLAB visualization function clarify that the size represents pixels, so I wondered if we should do the same, dunno about python visualization tool's conventions. 
Always better to specify the unit IMO instead of leaving it as an arbitrary/dimensionless thing.

**Comment by jeromedockes** [comment]
in practice since the user doesn't know the size of the figure (a dynamic 3d plot) it remains a dimensionless arbitrary thing. but i can say it is in pixels in the docstring to prevent the hope that it can be in mm

**Comment by kchawla-pi** [comment]
Makes sense since pixel size depends on image resolution and is not fixed

**Comment by kchawla-pi** [comment]
Fixed in #1899
